src/Resource.js

```diff
@@ -265,11 +257,20 @@ export default class Resource extends Component {
     } else if (components[type] != null) {
       componentToRender = components[type];
     } else {
+      const expanded = this.props.isCartridgeRemotelyExpanded;
+      const resource = this.props.resourceMap.get(this.props.identifier);
+      const path = getResourceHref(resource);
+      let filePath = undefined;
+      if (expanded) {
+        filePath = `${this.props.basepath}/${path}`;
+      }
```

Comment:
If I see correctly, the main issue was here, that the saveAs function is not able to handle the download with some browsers (got 404 for that resource). Right now it is switch between a simple href and the saveAs version based on the isCartridgeRemotelyExpanded. If expanded, we use the href, other way we use the saveAs magic.

What will be happened in Chrome, if the resource is not isCartridgeRemotelyExpanded?